openerp/tools/mail.py

```diff
@@ -116,6 +119,10 @@ def html_sanitize(src, silent=True, strict=False, strip_style=False, strip_class
         logger.warning('unknown error obtained when sanitizing %r', src, exc_info=True)
         cleaned = '<p>Unknown error when sanitizing</p>'
 
+    root = lxml.html.fromstring(cleaned)
+    sanitize_attributes(root)
+    cleaned = ustr(etree.tostring(root, encoding='utf-8'))
```

Comment:
IIRC lxml supports `encoding='unicode'` (or `encoding=unicode`), which should have the same behaviour except better (since it doesn't encode-then-decode)